src/manifest/mod.rs

```diff
@@ -36,6 +40,63 @@ struct CargoPackage {
     repository: Option<String>,
 }
 
+#[derive(Default, Deserialize)]
+struct CargoMetadata {
+    #[serde(rename = "wasm-pack")]
+    #[allow(dead_code)]
+    wasm_pack: CargoWasmPack,
+}
+
+#[derive(Default, Deserialize)]
+struct CargoWasmPack {
+    #[allow(dead_code)]
+    profile: CargoWasmPackProfiles,
+}
+
+#[derive(Deserialize)]
+struct CargoWasmPackProfiles {
+    #[allow(dead_code)]
+    dev: CargoWasmPackProfile,
+
+    #[allow(dead_code)]
+    release: CargoWasmPackProfile,
+
+    #[allow(dead_code)]
+    profiling: CargoWasmPackProfile,
+}
+
+impl Default for CargoWasmPackProfiles {
+    fn default() -> CargoWasmPackProfiles {
+        let dev = CargoWasmPackProfile::dev();
+        let release = CargoWasmPackProfile::release();
+        let profiling = CargoWasmPackProfile::profiling();
+        CargoWasmPackProfiles {
+            dev,
+            release,
+            profiling,
+        }
+    }
+}
+
+#[derive(Deserialize)]
+struct CargoWasmPackProfile {
+    // This is where wasm-opt, wasm-snip, etc configuration will go.
```

Comment:
As a proof of concept, could configuration for `wasm-bindgen` be added? Should allow passing flags like `--no-demangle` and `--keep-debug`